.jenkins/pytorch/test.sh

```diff
@@ -150,6 +150,10 @@ if [ -n "$IN_PULL_REQUEST" ] && [[ "$BUILD_ENVIRONMENT" != *coverage* ]]; then
   file_diff_from_base "$DETERMINE_FROM"
 fi
 
+if [[ "$BUILD_ENVIRONMENT" == *ppc64le* ]]; then
+  SUDO=sudo
+fi
```

Comment:
could you add a comment for this change.

Reply:
I thought I do need sudo permission to run the command (like https://github.com/pytorch/pytorch/blob/ab8af155455bc9148234b4a595a6e3280478478d/.jenkins/pytorch/test.sh#L200-L208) but it turns out that I don't. Removed this part